### ISSUE TITLE
Code samples should not be set to aria-live="polite" after core/highlight executes

### ISSUE THREAD

**Issue by majornista** [w3c/respec#1456]
This is a regression for a7394e3cd22513e873a706f6e3391d6fb6bff323, which presents a significant accessibility problem for screen reader users trying to read a spec containing a lot of code examples, because every code sample is sent to the the screen reader buffer to announce at its earliest convenience. 

I've sometimes been unable to test a page using VoiceOver with a W3C spec open in another tab, because all I hear are source code examples from the spec being announced.

**Comment by marcoscaceres** [comment]
Ah yeah, got wrong advice it seems. Will remove.
